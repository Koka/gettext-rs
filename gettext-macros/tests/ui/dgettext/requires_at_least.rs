use gettext_macros::*;

fn main() {
    dgettext!();
}
