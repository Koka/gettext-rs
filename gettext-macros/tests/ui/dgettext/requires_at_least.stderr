error: requires at least domainname and msgid string arguments
 --> tests/ui/dgettext/requires_at_least.rs:4:5
  |
4 |     dgettext!();
  |     ^^^^^^^^^^^
  |
  = note: this error originates in the macro `dgettext` (in Nightly builds, run with -Z macro-backtrace for more info)
